cnho_stables/assetlist.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cnhostables",
+  "assets": [
+    {
+      "description": "CNHO Stables Network is designed to facilitate the real world asset in Cosmos ecosystem. Starting from non USD stablecoins and  stocks to real estates, the goal is to provide a new tool for the under development regions to invest in the future and pay globally. The native token is CNHO, pegged 1:1 on CNH.",
+      "denom_units": [
+        {
+          "denom": "ucnho",
+          "exponent": 0
+        },
+        {
+          "denom": "CNHO",
+          "exponent": 6
+        }
+      ],
+      "base": "ucnho",
+      "name": "CNHO Stables",
+      "display": "CNHO",
+      "symbol": "CNHO",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.svg"
+      },
+     
+      "keywords": [
+        "CNHO"
+      ],
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.png",
+	  "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cnho_stables/images/chain.svg"
+          "theme": {
+            "primary_color_hex": "#708ffc"
+          }
+        }
+      ],
+      "socials": {
+        "website": "https://cnho.io/",
+        "discord": "https://discord.com/invite/bPmhYJc5rU",
```

Comment:
Discord and telegram are not accepted socials at this time

Reply:
ok,i remove it later